addons/hr_holidays/hr_holidays.py

```diff
@@ -390,7 +397,7 @@ def holidays_validate(self, cr, uid, ids, context=None):
             if record.holiday_type == 'employee' and record.type == 'remove':
                 meeting_obj = self.pool.get('calendar.event')
                 meeting_vals = {
-                    'name': record.name or _('Leave Request'),
+                    'name': record.display_name,
```

Comment:
@dmo-odoo Description field is not required so if it is set then we set value and if not set then we set like <Employee Name> on <Leave Type>.

Thanx